recipes/python_rc_dummy/meta.yaml

```
@@ -0,0 +1,25 @@
+{% set name = "python_rc_dummy" %}
+{% set version = "0.0.1" %}
```

As this is an "internal" package, I would start with an `_` like we have used for some other things, e.g. `_libgcc_mutex`. Also, we will probably never do releases here, thus we can keep a very simple version 1.
```suggestion
{% set name = "_python_rc" %}
{% set version = "1" %}
```